source/_integrations/sky_remote.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Sky Remote
+description: The Sky Remote integration allows you to control a Sky box with Home Assistant.
+ha_category:
+  - Remote
+ha_release:
+ha_domain: sky_remote
+ha_config_flow: true
+ha_codeowners:
+  - "@dunnmj"
+  - "@saty9"
+ha_iot_class: Assumed State
+ha_platforms:
+  - remote
+ha_integration_type: device
+related:
+  - docs: /docs/configuration/
+    title: Configuration file
+---
+
+The `sky_remote` {% term integration %} lets you control a Sky box using Home Assistant.
+
+## Supported models
+
+This integration is intended to control all Sky satellite receiver boxes with a LAN port. It will not control Sky stream pucks.
+
+## Configuration
+
+{% include integrations/config_flow.md %}
+
```

Comment:
please add a `{% configuration_basic %}` block, describing the input parameters to configure the integration